app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -193,15 +201,11 @@ export default function ExploreDisplayTabs({
     }
   }
 
-  if (!cellFaceting) {
-    getCellFacetingData(selectedCluster, selectedAnnot)
-  }
-
   // if the exploreParams update need to reset the initial cell facets
   useEffect(() => {
     const [newCluster, newAnnot] = getSelectedClusterAndAnnot(exploreInfo, exploreParams)
     getCellFacetingData(newCluster, newAnnot)
-  }, [exploreParams?.cluster, exploreParams?.annotation])
+  }, [exploreInfo?.cluster, exploreInfo?.annotation])
```

Comment:
Can you elaborate why this change was necessary?  I ask because I wonder if it explains some of the other "duplicate request" behavior we've seen other places, and if this paradigm could also be used there (like for intra-study gene search).

Reply:
Insightful question.  Per our chat, it turns out this fixed one problem, and created another.  I've opened SCP-5379 to track that fix.  For now, I've reverted back to `exploreParams`, which is noisier at a technical level but avoids user-facing confusing.  The ticket has further details and steps to reproduce.